File: unityProduct/Assets/Script/ViewController/example2ViewController.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using UnityEngine.UI;

public class example2ViewController : ViewController
{
    public Text lab = null;

    protected override void init()
    {
        base.init();
    }

    protected override void open(object obj)
    {
        base.open(obj);

        lab.text = obj.ToString();
    }
}

File: FgoProductCocos/assets/script/ViewController/example2ViewController.ts
import { _decorator, Component, Node } from 'cc';
import {ViewController} from './ViewController'

const { ccclass, property } = _decorator;

@ccclass('example2ViewController')
export class example2ViewController extends ViewController {
    
    protected init(): void {
        super.init();
    }

    protected open(obj: any): void {
        super.open(obj);
    }

    public close(): void {
        super.close();
    }

    protected onTimer(): void {
        super.onTimer();
    }

    public unitTest() :void
    {
        console.log("unitTest");
    }
}


